requirements.txt

```diff
@@ -1,2 +1,2 @@
-awscli==1.16.160
-pre-commit==1.15.2
+awscli==1.16.163
```

Comment:
@chrisgilmerproj Let's try to keep these pinned to the Homebrew versions. Right now the awscli is at 1.16.160. AFAICT, Homebrew only does updates divisible by 10 (1.16.150, 1.16.140, etc.).

Reply:
Oh, didn't know that! TIL. I haven't used homebrew for `awscli`. Maybe its time for me to switch.